test/cpp/interop/xds_interop_client.cc

```diff
@@ -288,18 +288,20 @@ class LoadBalancerStatsServiceImpl : public LoadBalancerStatsService::Service {
 };
 
 void RunTestLoop(std::chrono::duration<double> duration_per_query) {
-  std::vector<absl::string_view> rpc_methods = absl::StrSplit(FLAGS_rpc, ',');
+  std::vector<absl::string_view> rpc_methods =
```

Comment:
Why is this necessary? The input shouldn't have `,,`, so there's no empty string to skip.

And was this the reason for the failure?

Reply:
you are right, this one has a default value of UnaryCall so it's not necessary; i just added it in case the default for the flag changes in the future.  there is no harm in havin git